src/components/Widgets/GitHubTrending.vue

```diff
@@ -77,11 +77,11 @@ export default {
           slug: repo.repo,
           desc: repo.desc,
           lang: repo.lang,
-          link: repo.repo_link,
+          link: "https://github.com" + repo.repo,
```

Comment:
Double quotes will fail the linter. Also, we can can use string interpolation.

```diff
- link: "https://github.com" + repo.repo,
+ link: `https://github.com/${repo.repo}`,
```

Reply:
fixed